samples/getting-started/simple-algorithms/DeutschJozsa.qs

```diff
@@ -0,0 +1,131 @@
+                // the last qubit also, but this would not affect the final outcome.
+                // We use a within-apply block to ensure that the Hadamard transform is
+                // correctly inverted.
+                ApplyToEachA(H, queryRegister);
+            } apply {
+                // We now apply Uf to the n + 1 qubits, computing |𝑥, 𝑦〉 ↦ |𝑥, 𝑦 ⊕ 𝑓(𝑥)〉.
+                Uf(queryRegister, target);
+            }
+
+            // The following for-loop measures all qubits and resets them to
+            // zero so that they can be safely returned at the end of the
+            // using-block.
+            let resultArray = ForEach(MResetZ, queryRegister);
+
+            // Finally, the last qubit, which held the 𝑦-register, is reset.
+            Reset(target);
+
+            // We use the predicate `IsResultZero` from Microsoft.Quantum.Canon
+            // and compose it with the All function from
+            // Microsoft.Quantum.Arrays. This will return
+            // `true` if the all zero string has been measured, i.e., if the function
+            // was a constant function and `false` if not, which according to the
+            // promise on 𝑓 means that it must have been balanced.
+            return All(IsResultZero, resultArray);
+        }
+    }
+
+
+
+    // As before, we define an operation and a function to construct black-box
+    // operations and a test case to make it easier to test Deutsch–Jozsa
+    // algorithm from a C# driver.
+    internal operation BooleanFunctionFromMarkedElementsImpl(n : Int, markedElements : Int[], query : Qubit[], target : Qubit) : Unit {
+        // This operation applies the unitary
+
+        //     𝑈 |𝑧〉 |𝑘〉 = |𝑧 ⊕ 𝑥ₖ〉 |𝑘〉,
+
+        // where 𝑥ₖ = 1 if 𝑘 is an contained in the array markedElements.
+        // Operations of this form represent querying "databases" in
+        // which some subset of items are marked.
+        // We will revisit this construction later, in the DatabaseSearch
+        // sample.
+        for (markedElement in markedElements) {
+            // Note: As X accepts a Qubit, and ControlledOnInt only
+            // accepts Qubit[], we use ApplyToEachCA(X, _) which accepts
+            // Qubit[] even though the target is only 1 Qubit.
+            (ControlledOnInt(markedElement, ApplyToEachCA(X, _)))(query, [target]);
+        }
+    }
+
+
+    /// # Summary
+    /// Constructs an operation representing a query to a boolean function
+    /// 𝑓(𝑥⃗) for a bitstring 𝑥⃗, such that 𝑓(𝑥⃗) = 1 if and only if the integer
+    /// 𝑘 represented by 𝑥⃗ is an element of a given array.
+    ///
+    /// # Input
+    /// ## nQubits
+    /// The number of qubits to be used in representing the query operation.
+    /// ## markedElements
+    /// An array of the elements {𝑘ᵢ} for which 𝑓 should return 1.
+    ///
+    /// # Output
+    /// An operation representing the unitary 𝑈 |𝑧〉 |𝑘〉 = |𝑧 ⊕ 𝑥ₖ〉 |𝑘〉.
```

Comment:
My OS does not render the unicode symbol ₖ after the `x`. Maybe we can find a replacement?

Reply:
Does it not render in your browser, editor or neither?